docs/source/index.rst

```diff
@@ -21,7 +21,9 @@
 
     import pymc3 as pm
 
-    X, y = linear_training_data()
+    X = np.random.normal(size=100)
```

Comment:
Should we also add `import numpy as np`, or leave it just like this?

Reply:
I'd leave it out, because whoever ends up most probably knows what `np` is already

Reply:
Also you can't make code suggestions on unchanged lines, so someone would have to checkout out the branch and do it locally.
I'm too lazy for that.